awx/main/fields.py

```diff
@@ -514,10 +514,10 @@ def schema(self, model_instance):
             field = field.copy()
             if field['type'] == 'become_method':
                 field.pop('type')
-                field['choices'] = map(operator.itemgetter(0), CHOICES_PRIVILEGE_ESCALATION_METHODS)
+                field['choices'] = list(map(operator.itemgetter(0), CHOICES_PRIVILEGE_ESCALATION_METHODS))
```

Comment:
`map` returns an iterator. Maybe the cast to list can be dropped

PS: worth checking for more like this

Reply:
This was a case where our DRF code was doing list indexing and expected a list.